packages/x-components/src/x-installer/x-installer/__tests__/x-installer.spec.ts

```diff
@@ -188,6 +188,20 @@ describe('testing `XInstaller` utility', () => {
     expect(app?.$el).toHaveTextContent('test-2');
   });
 
+  it('should allow set the snippetConfig getter', () => {
```

Comment:
Instead of repeating the same test here, we should test that the x-installer is able to set the snppetConfig getter correctly. So I would test something like:

```
 it('should set the snippetConfig getter', async () => {
    const vue = createLocalVue();
    window.initX = getMinimumSnippetConfig();
    await new XInstaller({
      adapter,
      vue,
      app: createSnippetConfigComponent()
    }).init();
    const snippetConfig = window.InterfaceX?.getSnippetConfig();
    expect(snippetConfig).toEqual({
      instance: 'test',
      lang: 'test',
      scope: 'test',
      uiLang: 'test'
    });
  });
  ```